src/librustc_middle/ty/int.rs

```diff
@@ -0,0 +1,111 @@
+use crate::mir::interpret::truncate;
+use rustc_target::abi::Size;
```

Comment:
Now this file is even more removed from `ConstValue`, which it seems closely related to.^^

Maybe call this `ty/const.rs`, with the plan to move `ConstValue` here as part of https://github.com/rust-lang/rust/issues/72396?

Reply:
`const` isn't possible, but `consts`.